src/data/data.json

```diff
@@ -0,0 +1,324 @@
+{
+    "evaluationsTableData": 
+    {
+        "itemHeadings":
+        [
+            "Nume", "Tehnologie", "Nivel",""
+        ],
+        "items":
+        [
+            {
+                "name": "Popescu Adrian",
+                "technology": "Javascript",
+                "level": "Mid 2"
+            },
+            {
+                "name": "Dragan Roxana",
+                "technology": "PHP",
+                "level": "Mid 1"
+            },
+            {
+                "name": "Florescu Mihai",
+                "technology": "Javascript",
+                "level": "Junior 3"
+            },
+            {
+                "name": "Gheorghe Andrei",
+                "technology": "Ruby",
+                "level": "Senior 1"
+            }
+        ]
+    },
+
+    "technicalLevel": {
+        "title": "Technical level",
+        "headers": [
+            "Trainee", "Junior", "Middle", "Senior"
+        ],
+        "columnData": [
+            {
+                "inputValues": ["Trainee"]
+            },
+            {
+                "inputValues": [
+                    "Junior1", "Junior2", "Junior3"
+                ]
+            },
+            {
+                "inputValues": [
+                    "Middle1", "Middle2", "Middle3"
+                ]
+            },
+            {
+                "inputValues": [
+                    "Senior1", "Senior2", "Senior3"
+                ]
+            }
+        ]
+    },
+
+    "textarea": {
+        "textarea":
+        [
+            {
+                "label":"Should the candidate be hired?",
+                "placeholder": "The type of project that is suitable for the candidate&#13;&#10;Is guidance required for the candidate"
+            },
+            {
+                "label":"General Impression", 
+                "placeholder": "*required"
+            },
+            {
+                "label": "Workflow, Leadership &amp; Softskills",
+                "placeholder":  "Describe the environment in which the candidate works.&#13;&#10;Describe any guidance or management experience."
+            }
+        ]
+    },
+
+    "fieldset": {
+        "fieldset":
+        [
+            {
+                "legend":"OOP, Design Patterns",
+                "ul": [
+                    {
+                        "row": [
+                            {
+                                "label": "Classes",
+                                "options": ["0", "1", "2", "3"]
```

Comment:
Since we're repeating these `options` all over the place, it might be worthwhile considering abstracting these values away into a separate, generic location and reusing it wherever needed. That way, we'd have our config nice and DRY, as well as the rest of our code.